src/main/resources/META-INF/resources/primefaces/datatable/datatable.js

```diff
@@ -2399,6 +2399,12 @@ PrimeFaces.widget.DataTable = PrimeFaces.widget.DeferredWidget.extend({
         else {
             this.showRowEditors(row);
 
+            let inputs=$(row).find('input, select, textarea, button');
```

Comment:
I think a better selector would be `:input:enabled` which finds the first input,select,textarea radio button or any input element that is actually NOT disabled.